examples/fabric-chaincode-example-external/src/main/java/org/hyperledger/fabric/example/SimpleAsset.java

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
It would be preferable if we can use a Contract based example rather than a Chaincode example please.